calico_node/startup/startup.go

```diff
@@ -341,6 +341,22 @@ func validateIP(ipn *net.IPNet) {
 	warning("Unable to confirm IPv%d address %s is assigned to this host", ip.Version(), ip)
 }
 
+// evaluateENVBool evaluates a passed environment variable
+// Returns an boolean indicating if the parameter was defined
+// and an boolean if the argument was true or false
+// If it was false an additional log entry will be written
+
+func evaluateENVBool(envvar string, logentry string) (isDefined bool, outgoingNATEnabled bool) {
+	envValue, isEmpty := os.LookupEnv(envvar)
+	switch strings.ToLower(envValue) {
+	case "false", "0", "no", "n", "f":
+		log.Info(logentry)
+		return isEmpty, false
+	}
+
+	return isEmpty, true
```

Comment:
@caseydavenport Your example still shows 2 returned values though now we could get by with just one, do you think the isDefined return value should remain? I don't think so.

I had considered suggesting a change like this but had thought it allowed more flexibility because then the calling function could take a specific action if it was not defined.  That flexibility isn't really necessary here so I'm good with the change.  Plus with Casey's suggestion the feature to return if it is defined or not could always be added back.